includes/Plugin_Context.php

```diff
@@ -39,10 +41,18 @@ class Plugin_Context {
 	 *
 	 * @since 1.0.0
 	 *
+	 * @throws Exception Throws exception if not called via regular WP-CLI or WordPress bootstrap order.
+	 *
 	 * @param string $main_file The absolute path to the plugin main file.
```

Comment:
```suggestion
	 * @param string $main_file The absolute path to the plugin main file.
	 *
	 * @throws Exception Throws exception if not called via regular WP-CLI or WordPress bootstrap order.
```

Re-order for consistency